Branch-TestBed/Branch-TestBed.xcodeproj/project.pbxproj

```diff
@@ -208,7 +203,7 @@
 				ORGANIZATIONNAME = "Branch Metrics";
 				TargetAttributes = {
 					6700165F1940F51400A9E103 = {
-						DevelopmentTeam = 6B2DFWM843;
```

Comment:
What's the consequence of having this changed and committed? We've never paid attention to this value, and always let it be so far.